Tasks/FuncToolsInstallerV0/task.json

```diff
@@ -0,0 +1,50 @@
+{
+    "id": "A22A6EB4-FE21-442E-ABCE-5B2B09CFECE3",
+    "name": "FuncToolsInstaller",
+    "friendlyName": "Install Azure Func Core Tools",
+    "description": "Install Azure Func Core Tools",
+    "helpUrl": "https://aka.ms/func-tools-installer",
+    "helpMarkDown": "[Learn more about this task](https://aka.ms/func-tools-installer)",
+    "category": "Tool",
+    "visibility": [
+        "Build",
+        "Release"
+    ],
+    "author": "Microsoft Corporation",
+    "version": {
+        "Major": 0,
+        "Minor": 160,
+        "Patch": 0
+    },
+    "preview": true,
+    "demands": [],
+    "satisfies": [
+        "Func"
+    ],
+    "groups": [],
+    "inputs": [
+        {
+            "name": "version",
+            "label": "Version",
+            "type": "string",
+            "helpMarkDown": "Specify the version of Azure func tools to install. Ex:<br><br>2.7.1575<br>v2.7.1575<br>latest",
```

Comment:
Have you tried using `\n` instead of `<br>`

Reply:
No. `<br>` works fine.